api/server/router/network/network_routes.go

```diff
@@ -56,6 +56,10 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 		return err
 	}
 
+	if vars["id"] == "" {
+		return libnetwork.ErrNoSuchNetwork(vars["id"])
```

Comment:
We can just add an explicit route for the trailing slash:
```
router.NewGetRoute("/networks/", r.getNetworksList),
```

Displaying the bridge inspect for `/networks/` is a bug.